api/turing/cluster/servicebuilder/service_builder.go

```diff
@@ -128,6 +119,9 @@ type clusterSvcBuilder struct {
 	MaxMemory                 resource.Quantity
 	MaxAllowedReplica         int
 	TopologySpreadConstraints []corev1.TopologySpreadConstraint
+
+	// Knative service configs
+	knativeServiceConfig *config.KnativeServiceDefaults
```

Comment:
I've also refactored the `clusterSvcBuilder` struct a little to carry the `knativeServiceConfig` field, which in turn contains platform-level fields like the cpu/memory limit scaling factor. These are fields that are common to a single API server and do not need to be drilled all the way to each individual `KnativeService`.